README.md

```diff
@@ -38,7 +38,14 @@ This repository hosts an e-commerce project tailored for baby products. It is de
    ```bash
    cd {project_directory}
    python -m venv env
-   source env/bin/activate  # On Windows: `env\Scripts\activate`
+   ```
+   **On Windows:**
+   ```
```

Comment:
Specify Code Block Type Example:
```powersehll
<your_powershell cmd>
```

Reply:
I specified the code block type via '#' to highlight in color instead of '<>' for better reading purposes.